app/v1/models/response/AddStateBenefitsResponse.scala

```diff
@@ -32,8 +32,8 @@ object AddStateBenefitsResponse extends HateoasLinks {
       import data._
       Seq(
         listStateBenefits(appConfig, nino, taxYear),
-        updateStateBenefits(appConfig, nino, taxYear, benefitId),
-        deleteStateBenefits(appConfig, nino, taxYear, benefitId)
+        updateStateBenefit(appConfig, nino, taxYear, benefitId),
```

Comment:
the endpoint is **Update state benefits**

Reply:
There are two endpoints, Update State Benefit, and Update State Benefit Amounts